libnmstate/nm/sriov.py

```diff
@@ -43,11 +90,67 @@ def create_setting(iface_state, base_con_profile):
         if not sriov_setting:
             sriov_setting = nmclient.NM.SettingSriov.new()
 
+        vfs_config = sriov_config.get(Ethernet.SRIOV.VFS_SUBTREE, [])
+        vf_object_ids = {vf.get_index() for vf in sriov_setting.props.vfs}
+        vf_config_ids = {
+            vf_config[Ethernet.SRIOV.VFS.ID] for vf_config in vfs_config
+        }
+        _edit_sriov_vfs_in_setting(
```

Comment:
I think you can call first remove and then create to do the same.

Reply:
As we need to remove the in order to edit. First, I am removing all the vf_objects and then creating all the vfs from the config. Thanks!